doc/cephfs/file-layouts.rst

```diff
@@ -201,15 +201,27 @@ directories do not have layouts set:
 
     
 Adding a data pool to the MDS
----------------------------------
+-----------------------------
 
 Before you can use a pool with CephFS you have to add it to the Metadata Servers.
 
 .. code-block:: bash
 
     $ ceph fs add_data_pool cephfs cephfs_data_ssd
-    # Pool should now show up
-    $ ceph fs ls
+    $ ceph fs ls  # Pool should now show up
     .... data pools: [cephfs_data cephfs_data_ssd ]
 
 Make sure that your cephx keys allows the client to access this new pool.
+
+You can then update the layout on a directory in CephFS to use the pool you added:
+
+.. code-block:: bash
+
+    $ mkdir /mnt/cephfs/myecdir
+    $ setfattr -n ceph.dir.layout.pool -v cephfs_data_ssd /mnt/cephfs/myecdir
```

Comment:
Unrelated to this doc change -- but do we need a process to auto-tag the data pool as being owned by CephFS when this attribute is set?

Reply:
Nope, somebody called "Jason" already updated add_data_pool's implementation to add the tag.  Smart guy!

Reply:
Thank goodness -- just wanted to ensure there was no magic backdoor of config settings.

Reply:
I think you should reconcile the original doc using ``cephfs_data_ssd`` pool name with your intent here to document EC pools via the ``myecdir``folder name.
Pick one, or duplicate for both?

Reply:
Fair point, head was in two places.  Updated.